SportsGoods.Api/Program.cs

```diff
@@ -1,25 +1,50 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using SportsGoods.App.Queries;
+using SportsGoods.App.QueryHandlers;
+using SportsGoods.Core.Interfaces;
+using SportsGoods.Data.DAL;
+using SportsGoods.Data.Repositories;
+using System.Reflection;
```

Comment:
Remove unused 'using'(s)

<img width="272" alt="image" src="https://github.com/Sports-Goods-B2B/SportsGoods/assets/10927289/4915bfb0-f84f-4c9d-ac3a-a04c1caa1346">

Ctrl+R+G in VS